airbyte-integrations/connectors/source-opsgenie/unit_tests/test_stream.py

```diff
@@ -0,0 +1,508 @@
+            json={
+                "totalCount": 8,
+                "data": [
+                    {
+                        "blocked": False,
+                        "verified": False,
+                        "id": "b5b92115-bfe7-43eb-8c2a-e467f2e5ddc4",
+                        "username": "john.doe@opsgenie.com",
+                        "fullName": "john doe",
+                        "role": {"id": "Admin", "name": "Admin"},
+                        "timeZone": "Europe/Kirov",
+                        "locale": "en_US",
+                        "userAddress": {"country": "", "state": "", "city": "", "line": "", "zipCode": ""},
+                        "createdAt": "2017-05-12T08:34:30.283Z",
+                    },
+                    {
+                        "blocked": False,
+                        "verified": False,
+                        "id": "e07c63f0-dd8c-4ad4-983e-4ee7dc600463",
+                        "username": "jane.doe@opsgenie.com",
+                        "fullName": "jane doe",
+                        "role": {"id": "Admin", "name": "Admin"},
+                        "timeZone": "Europe/Moscow",
+                        "locale": "en_GB",
+                        "tags": ["tag1", "tag3"],
+                        "userAddress": {
+                            "country": "US",
+                            "state": "Indiana",
+                            "city": "Terre Haute",
+                            "line": "567 Stratford Park",
+                            "zipCode": "47802",
+                        },
+                        "details": {"detail1key": ["detail1dvalue1", "detail1value2"], "detail2key": ["detail2value"]},
+                        "createdAt": "2017-05-12T09:39:14.41Z",
+                    },
+                ],
+                "took": 0.261,
+                "requestId": "d2c50d0c-1c44-4fa5-99d4-20d1e7ca9938",
+            },
+        )
+
+        responses.add(
+            "GET",
+            "https://api.opsgenie.com/v2/users/b5b92115-bfe7-43eb-8c2a-e467f2e5ddc4/teams",
+            json={
+                "data": [{"id": "6fa6848c-8cac-4cea-8a98-ad9ff23d9b16", "name": "TeamName"}],
+                "took": 0.023,
+                "requestId": "bc40b7ad-11ee-4dcd-ae5f-6d75dbc16261",
+            },
+        )
+
+        responses.add(
+            "GET",
+            "https://api.opsgenie.com/v2/users/e07c63f0-dd8c-4ad4-983e-4ee7dc600463/teams",
+            json={
+                "data": [
+                    {"id": "6fa6848c-8cac-4cea-8a98-ad9ff23d9b16", "name": "TeamName"},
+                    {"id": "bc40b7ad-11ee-4dcd-ae5f-6d75dbc16261", "name": "TeamName"},
+                ],
+                "took": 0.023,
+                "requestId": "bc40b7ad-11ee-4dcd-ae5f-6d75dbc16261",
+            },
+        )
+
+        records = list(read_full_refresh(stream))
+        self.assertEqual(3, len(records))
+
+
+class AlertsStreamTestCase(unittest.TestCase):
```

Comment:
`Alerts` as per earlier comments

Reply:
Reintroduced alerts testing. No need to remove.